src/terra-dev-site/app/how-to.3/change-application-theme.app.mdx

```diff
@@ -2,7 +2,7 @@
 
 The application theme is set using the `themeName` prop in ApplicationBase. The `themeName` is used as the class name for applying styles.
 
-A `terra-theme.config.js` file must be available to enable themes. Check out the [theme strategy guide](http://engineering.cerner.com/terra-application/components/terra-application/application/how-to/theme-strategy) for more information about using and applying themes.
+A `terra-theme.config.js` file must be available to enable themes. Check out the [theme strategy guide](/application/terra-application/how-to/theme-strategy) for more information about using and applying themes.
```

Comment:
Was it found the terra-app links will not resolve correctly using this pattern or does this work as expected in both terra-application's site & terra-ui's site??

Reply:
From terra-ui navigating to this url `http://engineering.cerner.com/terra-application/components/terra-application/application/how-to/theme-strategy` will take you from the terra-ui site to the terra-application site when the same content is hosted on both. By making this url relative, you will remain on the terra-ui site. This will work for both though we don't intend people to directly use terra-application site, it's for dev purposes. terra-ui is our primary consumer facing site.